install/install.go

```diff
@@ -265,19 +446,23 @@ func K8sInstaller(c *k8s.Client, o Options) error {
 
 	verify = o.Verify
 	var env string
-	if o.Env.Auto {
-		env = k8s.AutoDetectEnvironment(c)
-		if env == "none" {
-			if o.Save {
-				printMessage("⚠️\tNo env provided with \"--save\", setting env to  \"generic\"", true)
-				env = "generic"
-			} else {
-				return errors.New("unsupported environment or cluster not configured correctly")
-			}
+	env = k8s.AutoDetectEnvironment(c)
```

Comment:
Can't we just roll back to the way it was earlier? I see a lot of edge case issues here like save won't work when k8s client is missing